### PR TITLE
Update lunr-server and team-api

### PR DESCRIPTION
**Pull request by mbland**
This PR breaks down to:
- Some more minor `./go` script tweaks
- Getting lunr-server to use the `/config` volume convention, and updating all other daemon containers to reference the `/config` directory. Mounting config volumes as `/usr/local/18f/team-api/config` instead of just `/usr/local/18f/team-api` proved necessary to avoid mounting a writable volume (`/usr/local/18f/team-api/team-api.18f.gov`) on top of a read-only volume (`/usr/local/18f/team-api`).
- Creating an SSH config volume that copies data to `$HOME/.ssh` and runs `ssh-agent` and `ssh-add`. This is necessary for the `team-api`, `hub`, and `pages` daemons to access private repositories.
  - I created a key for the 18f-bot account, and added the account as a collaborator to 18F/team-api.18f.gov and 18F/hub. Not sure if it's worth making the bot a read-only 18F org member, or if we need to tell everyone posting to pages-staging and pages-private that they need to add the 18f-bot as a read-only collaborator.
  - This hasn't been an issue before because the existing key associated with the Hub/Team API/Pages services is associated with my GitHub account.
- Ended up using an `ENTRYPOINT` with `team-api`. I'll go back and update all the other daemons to use this format in a future PR.

cc: @jcscottiii @ccostino @afeld

### CODE REVIEW

**Comment by mbland** [comment]
Whoops, forgot to cc: @catherinedevlin earlier.

File: go [hunk 1]
```diff
@@ -149,13 +162,20 @@ def_command :stop_daemons, 'Stop Docker containers running as daemons' do |args|
...
+def_command :rm_containers, 'Remove stopped (non-data) containers' do |args|
   containers_to_stop_regex = "(#{_images(args).join('|')})"
   exec_cmd 'containers=$(docker ps -a | sed -e \'s/.* \([^ ]*$\)/\1/\' | ' \
     "egrep -v '(NAMES|-data)$' | egrep -- '#{containers_to_stop_regex}'); " \
     'if [ ! -z "$containers" ]; then docker rm $containers; fi'
 end
 
+def_command(
+  :rm_all_containers, 'Remove all stopped (non-data) containers') do |args|
+  exec_cmd 'containers=$(docker ps -a | sed -e \'s/.* \([^ ]*$\)/\1/\' | ' \
```
Comment (jcscottiii) [inline go]: could you move `containers=$(docker ps -a | sed -e \'s/.* \([^ ]*$\)/\1/\'` into it's own function or variable. it's used above in 167 too. and maybe the egrep too if possible?

**Comment by jcscottiii** [comment]
@mbland i just had one comment. everything else looks good!

File: go (hunk 1 above)
Comment (mbland) [inline go]: Done! Made this and `rm_images` more ruby-istic, and eliminated `rm_all_containers`.

**Comment by mbland** [comment]
@jcscottiii The Travis build finally passed!

**Comment by mbland** [comment]
cc: @batemapf

**Pull request merged**